pkg/sentry/inet/inet.go

```diff
@@ -153,3 +153,21 @@ type Route struct {
 	// GatewayAddr is the route gateway address (RTA_GATEWAY).
 	GatewayAddr []byte
 }
+
+// Describes Ip line of /proc/net/snmp.
+type StatSnmpIp [19]uint64
```

Comment:
According to the style guide, this should be StatSNMPIP. https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Same for the others.

Reply:
Done. Thanks!